src/DotVVM.Framework/Controls/GridViewColumn.cs

```diff
@@ -205,6 +210,28 @@ private void SetSortedCssClass(HtmlGenericControl cell, ISortableGridViewDataSet
             // TODO: verify that sortExpression is a single property name
             return SortExpression;
         }
+
+        [ControlUsageValidator]
+        public static IEnumerable<ControlUsageError> ValidateUsage(ResolvedControl control)
+        {
+            if (control.Properties.ContainsKey(DataContextProperty))
+            {
+                yield return new ControlUsageError("Changing the DataContext property on the GridViewColumn is not supported!", control.DothtmlNode);
+            }
+
+            // disallow attached properties on columns
+            foreach (var property in control.Properties)
+            {
+                // ignore attached properties that are set by runtime and not from markup
+                if (property.Value.DothtmlNode == null) continue;
```

Comment:
This will also allow setting these properties by server-side styles, not sure we want that.

Reply:
It will also disallow the Validator.Value that we support, right?

Reply:
@exyi `Validator.Value` is set automatically to the same expression as the `ValueBinding`, so there is no point in setting `Validator.Value` on the column. If someone wants to do it, we'll have `CellDecorators` for that.